README.md

```diff
@@ -90,6 +93,8 @@ above.
   1. edit `/etc/openstack_deploy/user_extras_variables.yml` to add credentials
   2. run the MaaS setup plays:
      `cd /opt/rpc-openstack/rpcd/playbooks && openstack-ansible setup-maas.yml`
+  3. run the MaaS verify play:
```

Comment:
That may be. I just don't want us to trigger our "more and more verbose as retries number gets higher" behaviour for this stuff